docs/docs/98-contributing/03-debugging.md

```diff
@@ -0,0 +1,22 @@
+# Debugging
+
+To debug the code generated by Mapperly just set a breakpoint in the generated code and you are happy to go.
```

Comment:
Is "happy to go" a common expression? I only know about "ready to go"

Reply:
replaced with ready to go